matlab/scripts/correlation_value.m

```diff
@@ -9,6 +9,8 @@
 %% Load in files
```

Comment:
Not sure what this code is doing - I don't see it calculating a correlation value, which is what the file name suggests. You should rename the file name, and clean up the rest of the script.

Reply:
Thats the old one (one of the ones I might just delete)